backend/app/views/reports/report_content.erb

```diff
@@ -4,7 +4,7 @@
 <% xml_clean!(info) if report.format == 'pdf' %>
 
 <div class="header">
-  <img class="logo" src="<%= @base_url || '/' %>reports/static/images/archivesspace.small.png" alt="ArchivesSpace logo"/>
+  <img class="logo" src="<%= @base_url || '/' %>reports/static/images/ArchivesSpaceLogo.svg" alt="ArchivesSpace logo"/>
```

Comment:
@brianzelip As best I can tell this is doing nothing and has been doing nothing. There is no setting of `@base_url` that I can find, and without it I don't see how the image path would be resolvable in PDF or HTML exports. Maybe this line of code was effectively stranded by this commit in 2018?
https://github.com/archivesspace/archivesspace/commit/a18241ead0ca180021622506d8917ed919d122e8#diff-55e2bcb50cc5fd6f5bb38e1b8cce6d865234a82e49e5bbeaa5502cb7647358ee